sklearn/decomposition/tests/test_pca.py

```diff
@@ -656,3 +659,41 @@ def test_assess_dimesion_rank_one():
     assert np.isfinite(_assess_dimension(s, rank=1, n_samples=n_samples))
     for rank in range(2, n_features):
         assert _assess_dimension(s, rank, n_samples) == -np.inf
+
+
+def test_pca_svd_output():
```

Comment:
We should only add a non-regression test as the one provided in the issue to ensure that increasing this number will increase the precision.